client/src/services/formatting.js

```diff
@@ -5,6 +5,17 @@ export const formatString = (string, params) => {
     return string;
 };
 
-export const getVideoIdFromLink = (videoLink) => {
-    return videoLink.split("?v=")[1];
-};
\ No newline at end of file
+export const paramsObjectToQueryString = paramsObject => {
```

Comment:
Try to read about the built in JavaScript classes `URL` and `URLSearchParams`
 (don't be a @NIV54i and implement `DateTime` on your own)